ubuntu_image/helpers.py

```diff
@@ -117,7 +117,7 @@ def snap(model_assertion, root_dir, channel=None, extra_snaps=None):
 
 
 def sparse_copy(src, dst, *, follow_symlinks=True):
-    args = ['cp', '-p', '--sparse=always', src, dst]
+    args = ['cp', '--preserve=mode,ownership', '--sparse=always', src, dst]
```

Comment:
It bears thinking a bit more deeply about whether -p buys us anything at all or if it should be dropped completely (that deep thinking is why I hadn't raised an MP for this myself yet).

The file that we're copying is one that we've created, without explicitly setting permissions; it should therefore be owned by the process's uid/gid and have file permissions according to the process umask.  Calling 'cp' should give us the exact same thing for the target file.  So is there any reason not to just drop -p?